uint8 GetPoint          = 1
uint8 DoorOpening       = 2
uint8 GotoPoint         = 3
uint8 GoalDescription   = 4
uint8 Guiding           = 5
uint8 BacktoPoint       = 6

uint8 NowTask
uint8 NextTask
bool FinishState
bool NeedHelp
